### PR TITLE
feat: query *all* sites and match on exact site name

### PR DESCRIPTION
**Pull request by jasikpark**
This fixes two bugs:
- Without `filter=all`, you only see sites that are created by your account, which might not include other sites in a Netlify Team
- The `/sites?name=${siteName}` filter apparently works in a greedy way, returning all sites that contain the passed `siteName` string in their name.
cc @IanVS

### CODE REVIEW

**Comment by IanVS** [comment]
The `dist` would need to be removed before this could be merged, but for now, we can pin to this SHA in our github workflows since it's there.

**Comment by IanVS** [comment]
@kamranayub what do you think about this one?  I'm sure @jasikpark can remove the checked-in build artifacts if you'd consider merging it.

**Comment by jasikpark** [comment]
Okay, rebased and removed the checked-in `dist` files in the commits

**Comment by jasikpark** [comment]
okay... *now* it's ready xD

File: index.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const waitForUrl = async (url, MAX_TIMEOUT) => {
   for (let i = 0; i < iterations; i += 1) {
     try {
       await axios.get(url);
+      console.log('Url fetch completed, done!');
```
Comment (IanVS) [inline index.js:19]: Did you mean to leave this console log?

Comment (jasikpark) [reply index.js:19]: Yes, I figured it'd be helpful to see whether the url was set, though maybe this is unneeded in the end

**Comment by kamranayub** [comment]
Nice, let me see if I can update the npm token...

**Pull request merged**